integration-tests/rbac-permission.test.ts

```diff
@@ -0,0 +1,29 @@
+import { getFhirClient, randomPatient } from './utils';
+
+jest.setTimeout(60 * 1000);
+
+test('practitioner role can create new patient', async () => {
+    const client = await getFhirClient('practitioner');
+    const patientRecord: any = randomPatient();
+    delete patientRecord.id;
+    await expect(client.post('Patient', patientRecord)).resolves.toMatchObject({
+        status: 201,
+        data: patientRecord,
+    });
+});
+
+describe('Negative tests', () => {
+    test('invalid token', async () => {
+        const client = await getFhirClient('practitioner', 'Invalid token');
```

Comment:
Can we add more tests with invalid tokens? `Invalid token` gets rejected right away since it's syntactically invalid. I think we should add a couple more tests with syntactically valid tokens like:

- Invalid signature:`<valid header>.<valid claims. set exp to a date in the far distant future>.xxxInvalidSignaturexxx`
- Attempt to use an unsafe algorithm: `<valid header with alg=none>.<valid claims>` 

Actually these tests far more relevant for SMART since we own the token validation step. It's fair to assume that Cognito just works.